.github/workflows/cape-encrypt-release.yml

```diff
@@ -0,0 +1,85 @@
+name: Cape Encrypt Release
+
+on:
+  push:
+    tags:
+      - "[0-9]+.[0-9]+.[0-9]+"
```

Comment:
we probably want a separate way to trigger releasing this package, since it will not be tied to the versions of the other two packages